src/devices/Bmp280/Bmp280.cs

```diff
@@ -0,0 +1,562 @@
+        {
+            _i2cDevice.WriteByte((byte)Registers.REGISTER_CHIPID);
+            var readSignature = _i2cDevice.ReadByte();
+
+            if (readSignature != Signature)
+            {
+                return;
+            }
+            _initialized = true;
+
+            //Read the coefficients table
+            ReadCoefficients();
+        }
+
+        /// <summary>
+        ///  Reads the factory-set coefficients 
+        /// </summary>
+        private void ReadCoefficients()
+        {
+            // Read temperature calibration data
+            _calibrationData.DigT1 = Read16((byte)Registers.REGISTER_DIG_T1);
+            _calibrationData.DigT2 = (short)Read16((byte)Registers.REGISTER_DIG_T2);
+            _calibrationData.DigT3 = (short)Read16((byte)Registers.REGISTER_DIG_T3);
+
+            // Read pressure calibration data
+            _calibrationData.DigP1 = Read16((byte)Registers.REGISTER_DIG_P1);
+            _calibrationData.DigP2 = (short)Read16((byte)Registers.REGISTER_DIG_P2);
+            _calibrationData.DigP3 = (short)Read16((byte)Registers.REGISTER_DIG_P3);
+            _calibrationData.DigP4 = (short)Read16((byte)Registers.REGISTER_DIG_P4);
+            _calibrationData.DigP5 = (short)Read16((byte)Registers.REGISTER_DIG_P5);
+            _calibrationData.DigP6 = (short)Read16((byte)Registers.REGISTER_DIG_P6);
+            _calibrationData.DigP7 = (short)Read16((byte)Registers.REGISTER_DIG_P7);
+            _calibrationData.DigP8 = (short)Read16((byte)Registers.REGISTER_DIG_P8);
+            _calibrationData.DigP9 = (short)Read16((byte)Registers.REGISTER_DIG_P9);
+
+        }
+
+        /// <summary>
+        /// Set mode sleep (no measurements)
+        /// </summary>
+        public void SetModeSleep()
+        {
+            byte status = Read8((byte)Registers.REGISTER_CONTROL);
+            //clear last two bits
+            status = (byte)(status & 0b1111_1100);
+            _i2cDevice.Write(new[] { (byte)Registers.REGISTER_CONTROL, status });
+        }
+
+        /// <summary>
+        /// Sets mode to forced (device goes to sleep after one reading)
+        /// </summary>
+        public void SetModeForced()
+        {
+            byte status = Read8((byte)Registers.REGISTER_CONTROL);
+            //clear last two bits
+            status = (byte)(status & 0b1111_1100);
+            //set one of the last two bits
+            status = (byte)(status | 0b0000_0010);
+            _i2cDevice.Write(new[] { (byte)Registers.REGISTER_CONTROL, status });
+        }
+
+        /// <summary>
+        /// Sets mode to normal (continuous measurements)
+        /// </summary>
+        public void SetModeNormal()
+        {
+            byte status = Read8((byte)Registers.REGISTER_CONTROL);
+            //set last two bits to 11
+            status = (byte)(status | 0b0000_0011);
+            _i2cDevice.Write(new[] { (byte)Registers.REGISTER_CONTROL, status });
+        }
+
+        public PowerMode ReadPowerMode()
```

Comment:
>PowerMode [](start = 15, length = 9)

Should this just be named `ReadMode()` that way it will echo our `SetMode` methods. If not, then we could change the `SetMode` ones for `SetPowerMode` instead.